libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -638,6 +638,7 @@ int AP_HAL__I2CDevice_read_registers(lua_State *L) {
     return success;
 }
 
+#if HAL_GCS_ENABLED
```

Comment:
we should allow UART ops without GCS enabled
see my alternative PR with a web gui here:
https://github.com/ArduPilot/ardupilot/pull/25970
that PR also fixes networking ops from scripting, and socket ops

Reply:
the other PR https://github.com/ArduPilot/ardupilot/pull/25970 is merged

Reply:
Fixed